npm-package/jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  testEnvironment: 'node'
```

Comment:
I believe this file is unnecessary if the only option we are adding is the test environment key. I believe jest runs in the node test environment by default.